packages/firestore/src/lite-api/aggregate_types.ts

```diff
@@ -32,25 +33,30 @@ export class AggregateField<T> {
   /** A type string to uniquely identify instances of this class. */
   readonly type = 'AggregateField';
 
+  /** Indicates the aggregation operation of this AggregateField. */
+  readonly aggregateType: AggregateType;
+
   /**
    * Create a new AggregateField<T>
    * @param _aggregateType Specifies the type of aggregation operation to perform.
    * @param _internalFieldPath Optionally specifies the field that is aggregated.
    * @internal
    */
   constructor(
-    // TODO (sum/avg) make aggregateType public when the feature is supported
     readonly _aggregateType: AggregateType = 'count',
     readonly _internalFieldPath?: InternalFieldPath
-  ) {}
+  ) {
+    this.aggregateType = _aggregateType;
```

Comment:
the class now has both `this._aggregateType` and `this.aggregateType`.

Reply:
fixed.